htlcswitch/link.go

```diff
@@ -699,6 +700,21 @@ func (l *channelLink) syncChanStates() error {
 			"ChannelPoint(%v)", l.channel.ChannelPoint())
 	}
 
+	// If this is a tarpoot channel, then in addition to the normal reest
+	// message, we'll also send our local+remote nonces as well.
```

Comment:
`we'll also send our local+remote nonces as well` - I think this is left over from the previous version. We now only send our local verification nonce.